### ISSUE TITLE
¿Pull to refresh deshabilitado a propósito?

### ISSUE THREAD

**Issue by dresandev** [midudev/la-velada-web-oficial#415]
Hola, observo que hay tres propiedades que afectan el "pull to refresh" en dispositivos móviles. Mi pregunta es: ¿Están configuradas con alguna intención específica? ¿O puedo crear una PR corrigiendo el "pull to refresh"?

- archivo en el que se encuentran las propiedades y capturas de pantalla:

src/layouts/Layout.astro

![2](https://github.com/midudev/la-velada-web-oficial/assets/79766563/848a9bdb-dcf8-42ee-8c3e-704cf698c962)
![1](https://github.com/midudev/la-velada-web-oficial/assets/79766563/f0579e9a-7014-41ec-ba94-b2a5c810bf2d)

**Comment by MiguelHigueraDev** [comment]
Las propiedades fueron añadidas debido a que en dispositivos móviles a veces algunos elementos se salían de los límites y causaba que la página se viera mal. Quizás hay que buscar otra solución al problema en vez de usar esas propiedades.

**Comment by dresandev** [comment]
> Las propiedades fueron añadidas debido a que en dispositivos móviles a veces algunos elementos se salían de los límites y causaba que la página se viera mal. Quizás hay que buscar otra solución al problema en vez de usar esas propiedades.

Entiendo, ¿podrías poner capturas de los elementos que provocan el overflow o indicarme en qué sección los puedo encontrar? Por favor.

**Comment by MiguelHigueraDev** [comment]
Cuando intenté solucionar el problema de otra manera, era este el que causaba el overflow:

![image](https://github.com/midudev/la-velada-web-oficial/assets/133175356/1a15c578-fb3f-4cbd-8b43-392de82fbf0d)

(La imagen que está atrás del vídeo)

Probe a hacer la imagen más pequeña en dispositivos de menor tamaño, pero mi solución no arreglaba el problema completamente eso sí, en tablets seguía viéndose mal, por lo que habría que retocarla.

**Comment by MiguelHigueraDev** [comment]
Estuve revisando y ahora el problema sigue incluso después de cambiar el tamaño de la imagen, habrá que investigar más para encontrar la causa.

**Comment by dresandev** [comment]
> Estuve revisando y ahora el problema sigue incluso después de cambiar el tamaño de la imagen, habrá que investigar más para encontrar la causa.

Hola Miguel, por lo que veo, el problema de overflow en el componente src/sections/PresentationVideo.astro ya fue solucionado. En el mismo componente, el contenedor de los elementos tiene el overflow-hidden, solucionando el overflow que provocaba el componente DrawnXLogo.

![1](https://github.com/midudev/la-velada-web-oficial/assets/79766563/47b3827a-6008-469e-a6ce-50bb67d30d7f)

Quitando las propiedades que interferían con el "pull to refresh", el diseño luce como se espera.

https://github.com/midudev/la-velada-web-oficial/assets/79766563/62a06438-a82e-478b-a9cb-ff32e93556d1

Lo revisé en distintos dispositivos y navegadores, y no veo que haya algún otro problema. Sin embargo, mencionas que continúas observando el problema. ¿Podrías ser un poco más específico con el problema?

**Comment by MiguelHigueraDev** [comment]
> > Estuve revisando y ahora el problema sigue incluso después de cambiar el tamaño de la imagen, habrá que investigar más para encontrar la causa.
> 
> Hola Miguel, por lo que veo, el problema de overflow en el componente src/sections/PresentationVideo.astro ya fue solucionado. En el mismo componente, el contenedor de los elementos tiene el overflow-hidden, solucionando el overflow que provocaba el componente DrawnXLogo.
> 
> ![1](https://private-user-images.githubusercontent.com/79766563/311879290-47b3827a-6008-469e-a6ce-50bb67d30d7f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxOTc3NzQsIm5iZiI6MTcxMDE5NzQ3NCwicGF0aCI6Ii83OTc2NjU2My8zMTE4NzkyOTAtNDdiMzgyN2EtNjAwOC00NjllLWE2Y2UtNTBiYjY3ZDMwZDdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzExVDIyNTExNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjMzU1OTdjNjU2NzA4NGIyN2NhM2JhMGUyYjA3YWVhY2YwMzFhOGQ4MTc5ZTliZDg2ZWY2MmJlMjk0YjQxYzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9OqZ8DYsbzvhfqlmP7F37wagtV4umGSrhYLVK5WFWh8)
> 
> Quitando las propiedades que interferían con el "pull to refresh", el diseño luce como se espera.
> 
>  2.mp4 
> Lo revisé en distintos dispositivos y navegadores, y no veo que haya algún otro problema. Sin embargo, mencionas que continúas observando el problema. ¿Podrías ser un poco más específico con el problema?

Sí, disculpa, error mío. Eliminando el overflow-x-hidden de `Layout.astro` se soluciona el problema para mí igualmente.

![image](https://github.com/midudev/la-velada-web-oficial/assets/133175356/9face61a-c545-4e88-b0bb-37a3e9854c44)

Deberías hacer la pull request modificando Layout para eliminar la propiedad `overflow-x-hidden`. Gracias por tu colaboración.

**Comment by AlejandroSuero** [comment]
Creo que esta issue ya puede cerrarse con #445

**Comment by AlejandroSuero** [comment]
Si se deja el cambio que se ha hecho para el `pull-to-refresh`, se rompe el responsive.

@dresandev Mira a ver si puedes abrir otra PR y solucionarlo de otra manera.
